versioned_docs/version-latest/protocol/security.md

```diff
@@ -1,86 +1,266 @@
 ---
-title: Safety and Security
-sidebar_position: 6
-description: Security and Scam Prevention
+title: General and server security guidelines
```

Comment:
These changes are reflected in your other PR, let's keep the improvements separate and remove the changes from this PR, please :)